### PR TITLE
Added missing entry for crosswalk-app-tools.

### PR DESCRIPTION
**Pull request by jondong**
Also fixed the line ending issue in apis/_data.json.

### CODE REVIEW

**Comment by jondong** [comment]
@baleboy @chuan9 PTAL, thanks

File: public/documentation/_data.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
     "title": "开始",
     "origin": "getting_started"
   },
+  "crosswalk-app-tools": {
```
Comment (chuan9) [inline public/documentation/_data.json]: Are we sure that put the crosswalk-app-tools into the left navigation bar for english verison?

Comment (jondong) [inline public/documentation/_data.json]: Yes I believe so, and it seems that we forget to add an entry here for english version when the contents get submitted.

**Comment by chuan9** [comment]
LGTM, it's very helpfull, Thanks

**Pull request merged**